src/Icon/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import styles from './Icon.scss';
+
+function Icon(props) {
+  return (
+    <div>
+      <span
+        id={props.id}
+        className={classNames(props.className)}
+        aria-hidden={props.hidden}
+      />
+      { props.srText &&
+        <span className={classNames(styles['sr-only'])}>
+          {props.srText}
+        </span>
+      }
+    </div>
+  );
+}
+
+Icon.propTypes = {
+  id: PropTypes.string.isRequired,
+  className: PropTypes.arrayOf(PropTypes.string).isRequired,
+  hidden: PropTypes.bool,
+  srText: PropTypes.string,
+};
+
+Icon.defaultProps = {
+  hidden: true,
+  srText: undefined,
```

Comment:
The documentation makes it really clear what `srText` is supposed to represent (which is great). However, I'm wondering if the added clarity of the prop being called `screenReaderText` is worth the cost of a slightly more verbose name.

Reply:
This is a fair point, I think I defaulted this way because of the class name of `sr-only`, but the more verbose name here will definitely help with understanding for folks not immediately familiar with a11y rules.